.github/workflows/pull_request.yml

```diff
@@ -147,7 +147,7 @@ jobs:
       - name: Prepare CI Environment
         uses: ./.github/actions/prepare_env
       - name: Swift Build
-        run: "swift build --build-tests --disable-sandbox"
+        run: "swift build --disable-sandbox"
```

Comment:
this change doesn't do what the commit description says; this just _stops building tests_ which isn't what we want.

Reply:
Yes, the reason to build here is such that building is in a separate CI step than the testing; so we can easily spot if a failure was "during building" or "during testing"